pyproject.toml

```diff
@@ -35,6 +35,9 @@ select = [
     "F",
     "PIE810",
     "SIM101",
+    "SLOT000",
+    "SLOT001",
+    "SLOT002",
```

Comment:
You can just add the rule "SLOT" to enable these three rules. 

Moreover, i think that with new versions of ruff you should write [tool.ruff.lint]. See [here](https://docs.astral.sh/ruff/configuration/). But i understand that this is not the purpose of the Pr ;)

Reply:
Sir , I tried doing that but I think we already have lint.select=[] at the original place so we might not need to use `[tool.ruff.lint]` .